lib/widgets/atoms/profile_header.dart

```diff
@@ -18,52 +16,44 @@ class ProfileHeader extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final ThemeData theme = Theme.of(context);
+    var backgroundImage = avatarUrl != null
+        ? NetworkImage(avatarUrl!) as ImageProvider<Object>
+        : const AssetImage(Assets.blankAvatar);
+
     return Padding(
-      padding: const EdgeInsets.all(Insets.paddingExtraLarge),
+      padding: const EdgeInsets.fromLTRB(Insets.paddingExtraLarge,
+          Insets.paddingExtraLarge, Insets.paddingExtraLarge, 0),
       child: Row(
         crossAxisAlignment: CrossAxisAlignment.center,
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: [
-          Flexible(
-            child: Padding(
-              padding: const EdgeInsets.all(Insets.paddingExtraSmall),
-              child: Text(
-                profileMessage,
-                textAlign: TextAlign.start,
-                softWrap: true,
-                style: theme.textTheme.titleLarge?.copyWith(
-                  color: theme.colorScheme.onSurface,
-                ),
-              ),
```

Comment:
Without this Flexible widget there is risk of overflow. See this example:

<img width="344" alt="Screenshot 2023-09-28 at 6 58 51 PM" src="https://github.com/micromentor-team/mm-flutter-app/assets/25093428/1bc2a476-2346-422b-ae6d-95e33d0318f9">